### ISSUE TITLE
UI Design sync between Kompakkt and Semantic Kompakkt

### ISSUE THREAD

**Issue by lozanaross** [Kompakkt/Repo#40]
@Grizzly127  to coordinate work with @vmalieske and @NadNo12  that ensures Kompakkt and Semantic Kompakkt use nearly same UI design apart from Wikibase-specific components. 

Use existing Google doc and Figma files to track the pages that need UI updates.

**Comment by Grizzly127** [comment]
### Updates for the Explore page:

this is the Explore page  for kompakkt: 
<img width="558" alt="Bildschirmfoto 2024-05-15 um 15 19 31" src="https://github.com/Kompakkt/Repo/assets/44752496/6c02d7c6-8106-4988-9399-3c4909bac473">

this for semantic-kompakkt right now: 
<img width="551" alt="Bildschirmfoto 2024-05-15 um 15 19 39" src="https://github.com/Kompakkt/Repo/assets/44752496/7b099683-20a8-424e-8742-8525c11f8d3c">

- the "sort by" is missing for semantic kompakkt
- remove the history button from the filter options in kompakkt and add it to the navigation bar like in semantic kompakkt
- add the New Object and for now for kompakkt also the New Collections button to the navigation bar 
- the New object button for semantic kompakkt should be blue as well
- change the fontsize from the button to 14 
- if possible change also the search options to font size 14
- in semantic kompakkt it also says "Search for objects and compilations", but we don't have the "compilations" here, only in kompakkt, so better write here "Search in objects" and in kompakkt better "Search in objects and collections" (or depending from the toggle function of the Objects and Collections)
- align the search options with the logo at the navigation bar
- gap between buttons, history button and "Profile"/"Logout": 8px 
- (Sorry, my Figma file doesn't show the filter arrow correctly right now, but it stays the same)
<img width="590" alt="Bildschirmfoto 2024-05-15 um 16 08 34" src="https://github.com/Kompakkt/Repo/assets/44752496/bae001ca-3ea6-4e02-a188-1c580660df7a">
<img width="704" alt="Bildschirmfoto 2024-05-15 um 15 59 26" src="https://github.com/Kompakkt/Repo/assets/44752496/cb8150b8-bd76-4012-acff-ad584c131abf">

**Comment by lozanaross** [comment]
Hi Senya, I think this is a good start, but I think we should have the bullet list split in two groups - what needs changing in Kompakkt and what needs changing in Semantic Kompakkt. Maybe the SK bullets are only recorded in Gitlab, and here we only record what needs to change in Kompakkt. 

I also found this doc: https://docs.google.com/document/d/1LpYLtyvN7L9SqWqyrBQ2BbvKze5U57b1iUtAPg3_0ZM/edit should we start there and go page by page? I think it's good idea to start with Explore, but then maybe check the google doc and assign pages from that to individual colleagues to tackle, so we don't need to recreate evertything in comments here, or is the google doc already a bit outdated?

**Comment by vmalieske** [comment]
- add the New Object and for now for kompakkt also the New Collections button to the navigation bar

When the user is not logged in, there are buttons for Login and Registration. Should these also be removed (since Login and Registration are already in the Navbar)?

**Comment by vmalieske** [comment]
This points are done: 
- remove the history button from the filter options in kompakkt and add it to the navigation bar like in semantic kompakkt
 --> VE: Removed login and registration button from actionbar, removed actionbar from home-page
- add the New Object and for now for kompakkt also the New Collections button to the navigation bar
- change the fontsize from the button to 14
- if possible change also the search options to font size 14
- in semantic kompakkt it also says "Search for objects and compilations", but we don't have the "compilations" here, only in kompakkt, so better write here "Search in objects" and in kompakkt better "Search in objects and collections" (or depending from the toggle function of the Objects and Collections)
- align the search options with the logo at the navigation bar
- gap between buttons, history button and "Profile"/"Logout": 8px

Changes are within the pull request with the angular-material-upgrade.

**Comment by Grizzly127** [comment]
> Hi Senya, I think this is a good start, but I think we should have the bullet list split in two groups - what needs changing in Kompakkt and what needs changing in Semantic Kompakkt. Maybe the SK bullets are only recorded in Gitlab, and here we only record what needs to change in Kompakkt.

okay, yes I can do that. For me it's easier to have both next to each other to synchronize it best, but I can split it afterwards, so it's easier for @vmalieske to work on it. 

> I also found this doc: https://docs.google.com/document/d/1LpYLtyvN7L9SqWqyrBQ2BbvKze5U57b1iUtAPg3_0ZM/edit should we start there and go page by page? I think it's good idea to start with Explore, but then maybe check the google doc and assign pages from that to individual colleagues to tackle, so we don't need to recreate everything in comments here, or is the google doc already a bit outdated?

Yes, unfortunately it is outdated now. I can still use it as an orientation and to know which pages to look at, but most of it is outdated and was already somehow updated. At least for the Repo, I didn't look into the viewer yet. But that also means, that there is a lot of work done already.

**Comment by Grizzly127** [comment]
> * add the New Object and for now for kompakkt also the New Collections button to the navigation bar
> 
> When the user is not logged in, there are buttons for Login and Registration. Should these also be removed (since Login and Registration are already in the Navbar)?

Yes, @vmalieske I would remove them, you're right we already have it in the Navbar.

**Comment by Grizzly127** [comment]
### object options kompakkt
current options: 
<img width="1289" alt="Bildschirmfoto 2023-04-04 um 14 55 06" src="https://github.com/Kompakkt/Repo/assets/44752496/d8fba706-fad5-4650-8052-8400bb26f6bd">

- change the trash bin color to E91E63
- make sure the other parameter are correct; font: Open Sans, size: 14px
- object title same font and same size. (In kompakkt it was smaller than in semantic kompakkt.) both should also be 14px

**Comment by Grizzly127** [comment]
**Login and Register at the start page of kompakkt**
With the new Navigationbar the Login and Register options and the history button here on the start page of kompakkt should also be removed
<img width="1401" alt="Bildschirmfoto 2023-04-06 um 09 08 45" src="https://github.com/Kompakkt/Repo/assets/44752496/a1df9885-fe72-4e29-904a-c11b606f2256">

@vmalieske

**Comment by Grizzly127** [comment]
### Login and Register

**Register**
<img width="575" alt="Bildschirmfoto 2024-05-27 um 13 48 02" src="https://github.com/Kompakkt/Repo/assets/44752496/2fb65048-5aa6-4e61-8926-1fd9bb01303f">

- The font for the "Cancel" and "Create new account" is wrong, change it to Open Sans
- The "Cancel" button is orange, it should be pink: E91E63. 
- Same with the accent color of the input fields

**Login**
<img width="418" alt="Bildschirmfoto 2024-05-27 um 13 50 15" src="https://github.com/Kompakkt/Repo/assets/44752496/7fe48bc0-a98c-4183-b79f-312a71e68589">
Login is mostly synchronized, but the "Cancel" button here is black. I think we should use the same on like in the Registration. 
Also use pink as the accent color: E91E63. 

@lozanaross 
Question: I'm not sure with the colors anymore. Did we say we are using pink here and orange for semantic kompakkt or did we say we are using the same color for both? 

@vmalieske

**Comment by lozanaross** [comment]
> > Hi Senya, I think this is a good start, but I think we should have the bullet list split in two groups - what needs changing in Kompakkt and what needs changing in Semantic Kompakkt. Maybe the SK bullets are only recorded in Gitlab, and here we only record what needs to change in Kompakkt.
> 
> okay, yes I can do that. For me it's easier to have both next to each other to synchronize it best, but I can split it afterwards, so it's easier for @vmalieske to work on it.

This will work - thanks!
> 
> > I also found this doc: https://docs.google.com/document/d/1LpYLtyvN7L9SqWqyrBQ2BbvKze5U57b1iUtAPg3_0ZM/edit should we start there and go page by page? I think it's good idea to start with Explore, but then maybe check the google doc and assign pages from that to individual colleagues to tackle, so we don't need to recreate everything in comments here, or is the google doc already a bit outdated?
> 
> Yes, unfortunately it is outdated now. I can still use it as an orientation and to know which pages to look at, but most of it is outdated and was already somehow updated. At least for the Repo, I didn't look into the viewer yet. But that also means, that there is a lot of work done already.

OK, that's good news then.

**Comment by lozanaross** [comment]
> @lozanaross Question: I'm not sure with the colors anymore. Did we say we are using pink here and orange for semantic kompakkt or did we say we are using the same color for both?

I think we agreed that we'll use same color for both (so pink) when it comes to functional elements. Maybe we can discuss further how to use orange in SK in a way that only concerns the Wikibase metadata that doesn't exist in Kompakkt anyway. But that's a separate task. 

@HeyItsBATMAN can you assign this whole issue to @vmalieske so she gets notifications without need to be tagged in every comment.

**Comment by Grizzly127** [comment]
### object view option bar

<img width="1249" alt="Bildschirmfoto 2023-04-04 um 15 20 00" src="https://github.com/Kompakkt/Repo/assets/44752496/9ac2a864-177f-496a-b7e5-cefb4b20044d">

- Since we changed the navigation bar, the "History", "New Object" and "New Collection" button can be removed here.
- Move everything to the right to make it the same like in semantic kompakkt.
- We are using an "Annotate" button in semantic kompakkt, so replace the Annotate icon with the button. 
- change the color to blue for the "Annotate" and the "Publish" button. Colorcode: 00AFE7
- make sure the font of the buttons is Open Sans and size 14px. 

- I'm not sure about this icon:  ![image](https://github.com/Kompakkt/Repo/assets/44752496/cb1b11ad-57f7-49d1-aea5-77e0b3cac3ea) , it says "Use this object in a collection", I think it means to use it in an already existing collection. Is that option also possible in the "New Collection" button or can we integrate that there or should we keep the icon here? @lozanaross 

- The three icons under the picture description also are integrated in the option bar now to make it similar to semantic kompakkt.
- Align the navigation bar and the option bar (both with 24px gap from the right)
- gap between the icons: 16px, the rest stays 8px. 

<img width="883" alt="Bildschirmfoto 2024-06-06 um 17 49 21" src="https://github.com/Kompakkt/Repo/assets/44752496/21de46b3-89bb-4374-8486-a04a955c58a6">

**Comment by vmalieske** [comment]
To point 3 (Annotate-Button): How should the disabled button look like? At the moment, it is just grey:
![image](https://github.com/Kompakkt/Repo/assets/22995724/7e4865ba-ddcd-4fce-8947-61c2c1429e89)

And should there be a tooltip for that button?

**Comment by Grizzly127** [comment]
### Upload process 

Change the color of the "Reset" button here in kompakkt to the pink color 
And in previous versions sometimes there was a wrong font used (Helvetica Neue I think), I am not sure about the current font, it is not clear to me right now. But make sure for the buttons and for the text that Open Sans is used. 
<img width="868" alt="Bildschirmfoto 2024-06-12 um 15 10 07" src="https://github.com/Kompakkt/Repo/assets/44752496/8f4c1d44-25cc-4628-a7de-55e3385b86ca">

in kompakkt you can also see the annotation icon here, since you can't annotate here, we should remove it
<img width="1346" alt="Bildschirmfoto 2024-06-12 um 15 23 22" src="https://github.com/Kompakkt/Repo/assets/44752496/163ac988-e743-48de-bdf1-8baa7054661d">

The matadata formula looks very different, but I think this also belongs to the discussion of  #42 first.

something is wrong here at kompakkt at the Finlize site for the settings: 
<img width="878" alt="Bildschirmfoto 2024-06-12 um 15 32 13" src="https://github.com/Kompakkt/Repo/assets/44752496/2bd296b8-8275-4aca-b35f-9a24f7f661d1">

**Comment by lozanaross** [comment]
> To point 3 (Annotate-Button): How should the disabled button look like? At the moment, it is just grey: ![image](https://private-user-images.githubusercontent.com/22995724/338073326-7e4865ba-ddcd-4fce-8947-61c2c1429e89.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2MzM2MjUsIm5iZiI6MTcxODYzMzMyNSwicGF0aCI6Ii8yMjk5NTcyNC8zMzgwNzMzMjYtN2U0ODY1YmEtZGRjZC00ZmNlLTg5NDctNjFjMmMxNDI5ZTg5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE3VDE0MDg0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYyYmMwOWRhYTJkYzEwMTljMzZkNGY5YzU0Y2VhMTFjM2Y1ZWI3ZjFlYjU2ZTg5NWJlYmY0OGU5Y2IzOTQ3YWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.tOjvVGgiK6ZTMB4qgL5St8mqry5DPN41M_b7LAV9c7g)
> 
> And should there be a tooltip for that button? At the moment, there is a tooltip when disabled (You are not allowed to annotate right now) and no tooltip when enabled.

Thanks for the question @vmalieske - I think we don't need a tooltip when the button is active, it's self-explanatory. But I wonder why would we show it disabled. I think if the user doesn't have privileges to edit an object, the button should simply not be there? And it could appear when the user does have the rights. 

Or what do you think @HeyItsBATMAN @ZetOE @Grizzly127 

Alternatively, if the annotate button is showed grey, we should align the tooltip across both versions - currently in SK, it says: "Only the object owner is allowed to annotate this object." I don't think this is great, but neither is the Kompakkt version, we need some text to indicate that you can't annotate right now, but maybe you could if you add to collection. So perhaps we change the text in both versions to: "Add object to collection to be able to annotate"

**Comment by HeyItsBATMAN** [comment]
I don't have a strong opinion on this. I guess it can be removed if the user is not able to annotate it, but theoretically, a user can always annotate an object that is part of a collection, and any public object can be added to a collection, and then be annotated.

So I think the "Annotate"-button could be removed going forward, in favor of guiding the user to creating annotations inside of collections?

The only case where it's useful is if the uploader of an object needs to modify the default annotations, right?

**Comment by Grizzly127** [comment]
> We had a solution with a single icon that could solve both adding to an existing and to a new collection when we designed the new Explore / institutional page. Perhaps you can find that in the Figma file and paste here (I think the solution was in the pop up that opens when clicking the "[+]" icon). I think we should use that same icon here and position the collection-related icons as per your mockups:

Good idea, then we add this icon here and the user gets two options on adding an object to a collection in kompakkt: 

<img width="303" alt="Bildschirmfoto 2024-06-18 um 12 41 32" src="https://github.com/Kompakkt/Repo/assets/44752496/8b3589da-0a98-4cd2-b656-1795cfbf4c07">
<img width="544" alt="Bildschirmfoto 2024-06-18 um 12 42 06" src="https://github.com/Kompakkt/Repo/assets/44752496/02d875fa-eb8f-40f9-9bf1-d02107247e97">

**Comment by vmalieske** [comment]
@HeyItsBATMAN Could you have a look at the last commit? I had to rewrite the metadat-functionality (switch the icons from detail-page to option-bar) and want to be sure that it's okay.

**Comment by vmalieske** [comment]
> > We had a solution with a single icon that could solve both adding to an existing and to a new collection when we designed the new Explore / institutional page. Perhaps you can find that in the Figma file and paste here (I think the solution was in the pop up that opens when clicking the "[+]" icon). I think we should use that same icon here and position the collection-related icons as per your mockups:
> 
> Good idea, then we add this icon here and the user gets two options on adding an object to a collection in kompakkt:
> 

And then another small option-window for the existing collections?

**Comment by lozanaross** [comment]
> Good idea, then we add this icon here and the user gets two options on adding an object to a collection in kompakkt:

@Grizzly127 do you think we should move some buttons to the left? now all buttons are bundled on the right and maybe it's becoming a bit heavy (visually) how about if we move Publish & Annotate to the left (and then these buttons are not visible anyway when another user (not the object owner) opens an object.

**Comment by lozanaross** [comment]
> I don't have a strong opinion on this. I guess it can be removed if the user is not able to annotate it, but theoretically, a user can always annotate an object that is part of a collection, and any public object can be added to a collection, and then be annotated.
> 
> So I think the "Annotate"-button could be removed going forward, in favor of guiding the user to creating annotations inside of collections?

Maybe we can keep the Annotate button when an object is already inside a collection - @Grizzly127 do you want to work on a mockup of how this would look? it could just be the same button, but maybe there's also a special prompt once a user adds something in a collection in order to highlight the possibility for annotation a bit more. 

> 
> The only case where it's useful is if the uploader of an object needs to modify the default annotations, right?

Yes, I think the owner / uploader of an object should always see the Annotate button.

**Comment by vmalieske** [comment]
All design changes up to this point are in the pull request (forgot to mention the issue here).
Except the Annotate-issue, there I will wait for the mockup, right?

**Comment by Grizzly127** [comment]
> @Grizzly127 do you think we should move some buttons to the left? now all buttons are bundled on the right and maybe it's becoming a bit heavy (visually) how about if we move Publish & Annotate to the left (and then these buttons are not visible anyway when another user (not the object owner) opens an object.
> 
> And maybe the [+] button is after the edit button, because then Add, Embed, etc, are always visible, whereas Edit disappears when you're not logged in, or when you're not the owner.

Yes, we can change the edit and the [+] button, that makes sense. This is how it would look like to have the Publish and annotate on the left , but I think for me it was also okay to have all the functions in one place on the right.
<img width="804" alt="Bildschirmfoto 2024-06-21 um 17 07 16" src="https://github.com/Kompakkt/Repo/assets/44752496/409b85f4-88da-4ba5-b718-e86006565aab">

@lozanaross

**Comment by Grizzly127** [comment]
> That could be looked like this: ![image](https://private-user-images.githubusercontent.com/22995724/341461029-e41034ce-8e34-4e25-87e1-236aa602c181.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5ODM0MjQsIm5iZiI6MTcxODk4MzEyNCwicGF0aCI6Ii8yMjk5NTcyNC8zNDE0NjEwMjktZTQxMDM0Y2UtOGUzNC00ZTI1LTg3ZTEtMjM2YWE2MDJjMTgxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIxVDE1MTg0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYwZjgyZjQxYzg3NzUzYzc5YzA2YzliYWIzZmExMjNlNzgwMGQyMTY0Y2Q2ODNkZjVkZGIxMmExMTMxNmFkZmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pLUvKd2viD9vmZ95OI_J73JTKorgHBbN7fzL2HmbdW8)

For me this would be good for now to see the names of the collection where you can add it too, yes!:) 
I would just say the option is called: "add to existing collection" 

@vmalieske @lozanaross

**Comment by Grizzly127** [comment]
> > I don't have a strong opinion on this. I guess it can be removed if the user is not able to annotate it, but theoretically, a user can always annotate an object that is part of a collection, and any public object can be added to a collection, and then be annotated.
> > So I think the "Annotate"-button could be removed going forward, in favor of guiding the user to creating annotations inside of collections?
> 
> Maybe we can keep the Annotate button when an object is already inside a collection - @Grizzly127 do you want to work on a mockup of how this would look? it could just be the same button, but maybe there's also a special prompt once a user adds something in a collection in order to highlight the possibility for annotation a bit more.
> 
> > The only case where it's useful is if the uploader of an object needs to modify the default annotations, right?
> 
> Yes, I think the owner / uploader of an object should always see the Annotate button.

I don't know exactly what you mean here. Should I do a mock-up without the Annotate-button or a different kind of Annotate-button? Can you explain this issue again, please? 

@HeyItsBATMAN @lozanaross

**Comment by lozanaross** [comment]
> Yes, we can change the edit and the [+] button, that makes sense. This is how it would look like to have the Publish and annotate on the left , but I think for me it was also okay to have all the functions in one place on the right. 

Thanks @Grizzly127 - agreed - Publish + Annotate buttons look not too bad on the left, but they can also stay in one place on the right. Please proceed with whichever version you prefer.

**Comment by lozanaross** [comment]
> For me this would be good for now to see the names of the collection where you can add it too, yes!:) I would just say the option is called: "add to existing collection"

Yes, I also agree

**Comment by lozanaross** [comment]
> I don't know exactly what you mean here. Should I do a mock-up without the Annotate-button or a different kind of Annotate-button? Can you explain this issue again, please?

As discussed today, let's keep Annotate button always visible for owners of objects and when the object is already inside a collection. When the object is not in a collection and the user is not the owner, they should see a grey button with a tooltip over the button that says: "To be able to annotate, add this object to one of your collections."

**Comment by Grizzly127** [comment]
> Thanks @Grizzly127 - agreed - Publish + Annotate buttons look not too bad on the left, but they can also stay in one place on the right. Please proceed with whichever version you prefer.

Since the amount of buttons on the right, change sometimes when you are able to edit or not. Maybe it's better to put the Publish and Annotate button on the left
<img width="707" alt="Bildschirmfoto 2024-07-23 um 12 17 14" src="https://github.com/user-attachments/assets/44029e25-d73b-48f1-b5e1-7f659e4c3112">

> As discussed today, let's keep Annotate button always visible for owners of objects and when the object is already inside a collection. When the object is not in a collection and the user is not the owner, they should see a grey button with a tooltip over the button that says: "To be able to annotate, add this object to one of your collections."
<img width="676" alt="Bildschirmfoto 2024-07-23 um 12 18 31" src="https://github.com/user-attachments/assets/430c52b8-1e91-4221-883b-70f94048d4d4">

I'm not sure about the size of the tooltip, what do you think?
<img width="677" alt="Bildschirmfoto 2024-07-23 um 12 22 12" src="https://github.com/user-attachments/assets/d2dac767-9e4d-4390-86ad-c0ab34985366">

@vmalieske @lozanaross

**Comment by Grizzly127** [comment]
### Visibility Settings

These are the current settings for the visibility of an object in kompakkt
<img width="480" alt="Bildschirmfoto 2024-07-23 um 12 33 12" src="https://github.com/user-attachments/assets/e5aee837-fe00-4c86-bb29-bed6ead6d72d">

- [ ] Change the order of "Submit" and "Cancel" button
- [ ] Change the colors of the buttons
- [ ] make the _you_ in "Only **you** can see this object" bold. 

<img width="378" alt="Bildschirmfoto 2024-07-23 um 12 45 49" src="https://github.com/user-attachments/assets/bfd54343-0768-4452-a4be-1cd5f1dfcb98">


In semantic kompakkt only the buttons need to be changed, then it is already the same. 
What do you think about making it look cleaner already? 
For example: 
<img width="483" alt="Bildschirmfoto 2024-07-23 um 12 51 31" src="https://github.com/user-attachments/assets/6bcdcf9c-41bc-4bc8-abe0-ca36f596e6d9">

@vmalieske @lozanaross

**Comment by Grizzly127** [comment]
### History display

The history looks still different 
in kompakkt the icons are black and the gap is smaller: 
<img width="376" alt="Image" src="https://github.com/user-attachments/assets/e2a8d00b-02a9-40e2-bceb-8be9b7e18546" />

in sk the icons are grey and the gap is bigger: 
<img width="303" alt="Image" src="https://github.com/user-attachments/assets/79bcdbe3-2f15-4d8a-a995-967f9f38d16b" />

@vmalieske
